tests/unittests/test_runner.py

```diff
@@ -0,0 +1,107 @@
+from unittest import mock
+
+import pytest
+import requests
+
+from scenario_player.runner import ScenarioRunner, TokenNetworkDiscoveryTimeout
+
+
+@pytest.fixture
+def response_class():
```

Comment:
Nitpick: AFAICS this doesn't need to be a fixture

Reply:
True, true.